source/clear-linux/concepts/telemetry-about.rst

```diff
@@ -0,0 +1,82 @@
+.. _telemetry-about:
+
+Telemetrics
+###########
+
+One of the key features of |CLOSIA| is telemetry, which is used to
+monitor system health. Telemetry enables developers to observe and proactively
+address issues before end users are impacted.
+
+*Telemetrics* is a combination word made from:
+
+*	*Telemetry* which is sensing and reporting data.
+*	*Analytics* which is using visualization and statistical inferencing to make
+	sense of the reported data.
+
+|CL| telemetry reports system-level debug/crash information using specialized probes. The
+probes monitor system tasks such as :abbr:`swupd (software updater)`, kernel
+oops, machine error checks, and BIOS error report table for unhandled hardware
+failures. Telemetry enables real-time issue reporting to allow system
+developers to quickly focus on an issue and monitor corrective actions.
+
+|CL| telemetry is fully customizable and can be used during software development
+for debugging purposes. You can use **libtelemetry** in your code to create custom
+telemetry records. You can also use **telem-record-gen** in script files or call
+it from another program.
+
+Architecture
+************
+
+|CL| telemetry has two fundamental components, which are shown in figure 1:
+
+*	Client:  generates and delivers records to the backend server via the network.
+*	Backend: captures records sent from the client and displays the cumulative
+	content through a specialized interface.
+
+	.. note::
+
+		If you want to capture your own records for analysis, you must set up
+		your own backend server.
+
+.. figure:: figures/telemetry-about-1.png
```

Comment:
@MCamp859 It appears there's a subdirectory `telemetry-about` below concepts>figures. 
However, that subdirectory is not included in the `.. figure:: ...` directive. 

I recommend making changes, in this order:
1. Move `telemetry-about-1.png` up one directory so it resides in`figures`.
2. Delete secondary subdirectory `telemetry-about`.

This should fix the problem of the broken image link. :) Please create a new PR for this.